--- Arduino.h	2013-05-12 21:36:35.690710234 +0200
+++ Arduino.h~	2013-05-12 21:37:15.885207734 +0200
@@ -102,10 +102,10 @@
 void analogReference(uint8_t mode);
 void analogWrite(uint8_t, int);
 
-unsigned long millis(void);
-unsigned long micros(void);
-void delay(unsigned long);
-void delayMicroseconds(unsigned int us);
+unsigned long millis(void) __attribute__ ((weak));
+unsigned long micros(void) __attribute__ ((weak));
+void delay(unsigned long) __attribute ((weak));
+void delayMicroseconds(unsigned int us) __attribute ((weak));
 unsigned long pulseIn(uint8_t pin, uint8_t state, unsigned long timeout);
 
 void shiftOut(uint8_t dataPin, uint8_t clockPin, uint8_t bitOrder, uint8_t val);
