--- system_ticker.h	2013-05-12 21:46:11.860724915 +0200
+++ system_ticker.h~	2013-05-12 21:20:16.525315223 +0200
@@ -7,6 +7,10 @@
 #define ENC_COUNTS              7
 #define ENC_VELOCITY			8
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
 void system_ticker_setup(void);
 uint32_t millis(void);
 uint32_t micros(void);
@@ -14,4 +18,8 @@
 void delayMicroseconds(uint16_t us);
 int16_t encoder_get(uint8_t whatbit);
 
+#ifdef __cplusplus
+}
+#endif
+
 #endif
